alchemist/alchemist-cognitive-agents/src/test/kotlin/it/unibo/alchemist/test/TestFeelsTransmission.kt

```diff
@@ -27,8 +29,9 @@ class TestFeelsTransmission<T, P> : StringSpec({
 
     "social contagion makes nodes evacuate despite they haven't directly seen the danger" {
         loadYamlSimulation<T, P>("social-contagion.yml").startSimulation(
+            steps = 20000,
```

Comment:
Yes, otherwise the test fails. You'll see a lot of changes like that in this PR, the reason is that pedestrians move slower now that they cannot compenetrate each other.